transform/mattermost-analytics/models/staging/mm_telemetry_prod/_mm_telemetry_prod__sources.yml

```diff
@@ -0,0 +1,200 @@
+
+      - name: config_timezone
+        description: The raw Server timezone configuration data sent to Mattermost by Telemetry-Enabled Servers.
+
+      - name: config_webrtc
+        description: The raw Server WebRTC configuration data sent to Mattermost by Telemetry-Enabled Servers.
+
+      - name: config_audit
+        description: The raw Server Audit configuration data sent to Mattermost by Telemetry-Enabled Servers.
+
+      - name: config_bleve
+        description: The raw Server Bleve configuration data sent to Mattermost by Telemetry-Enabled Servers.
+
+      - name: plugins
+        description: Contains the raw plugin configurations of a server at a given point in time. Represented by integer counts. No plugin names or details provided.
+
+      - name: permissions_general
+        description: Contains the raw general permissions for each server at a given point in time/on a specific date.
+
+      - name: permissions_system_scheme
+        description: Contains the raw system scheme permissions for each server's user type at a given point in time/on a specific date - system admin, team guest user, system user, team admin, channel admin, channel guest, and channel user
+
+      - name: permissions_team_schemes
+        description: Contains the raw team scheme permissions for each server's user type at a given point in time/on a specific date - team guest, channel admin, channel user, team admin, channel guest, and team user.
+
+      - name: activity
+        description: Contains raw server-level activity at a given point in time/on a specific date. Includes number of posts, channels, slash commands, api calls, bot accounts, etc.
+
+      - name: event
+        description: >
+          Contains raw user-level event data as a single row per event.
+          Source is mattermost webapp ([code](https://github.com/mattermost/mattermost-webapp/blob/master/packages/mattermost-redux/src/client/rudder.ts#L35))
+        tags: ['tracks', 'mattermost-webapp']
+
+      - name: tracks
+        description: > 
+          Contains raw user-level data from rudderstack track method. Note that for every `track` call, RudderStack
+          creates a record in both the `tracks` and `<event_name>` tables.
+
+      - name: identifies
+        description: Contains raw user-level data from rudderstack identifies method.
+
+      - name: pages
+        description: Contains raw user-level data from rudderstack pages method.
+
+      - name: _groups
+        description: Contains raw server-level group data as a single row per server per timestamp.
+
+      - name: channel_moderation
+        description: Contains raw server-level channel data as a single row per server per timestamp.
+
+      - name: elasticsearch
+        description: Contains raw server-level ElasticSearch data as a single row per server per timestamp.
+
+      - name: warn_metrics
```

Comment:
I would be removing this. While we still receive telemetry from older versions, the telemetry is not used anymore, feature/functionality has been cut.

Reply:
Perhaps there are other events like this. It would be easy to figure out as soon as we have the report in looker. I'd suggest to create a list of deprecated events as soon as we finish the current ticket, and then think whether we want to show them as deprecated on Looker or just remove them.